tools/image_processing/imagej2/imagej2_adjust_threshold_binary.xml

```diff
@@ -24,86 +24,119 @@ ImageJ --ij2 --headless --debug
             <output name="output" file="blobs_threshold_default.gif" compare="sim_size"/>
         </test>
         <test>
             <param name="input" value="blobs.gif"/>
-            <param name="threshold_min" value="118.0"/>
-            <param name="threshold_max" value="255.0"/>
-            <param name="method" value="IJ_IsoData"/>
+            <conditional name="method_cond">
+                <param name="method" value="Percentile"/>
+                <param name="black_background" value="no"/>
+            </conditional>
             <param name="display" value="over_under"/>
-            <param name="black_background" value="no"/>
-            <param name="stack_histogram" value="no"/>
-            <output name="output" file="blobs_threshold_ijiso.gif" compare="sim_size"/>
+            <output name="output" file="blobs_threshold_percentile.gif" compare="sim_size"/>
         </test>
         <test>
             <param name="input" value="blobs.gif"/>
-            <param name="threshold_min" value="72.0"/>
-            <param name="threshold_max" value="255.0"/>
-            <param name="method" value="Huang"/>
+            <conditional name="method_cond">
+                <param name="method" value="Huang"/>
+                <param name="black_background" value="yes"/>
+            </conditional>
             <param name="display" value="bw"/>
-            <param name="black_background" value="yes"/>
-            <param name="stack_histogram" value="no"/>
             <output name="output" file="blobs_threshold_huang_dark.gif" compare="sim_size"/>
         </test>
+        <test>
+            <param name="input" value="blobs.gif"/>
+            <conditional name="method_cond">
+                <param name="auto_method" value="Manual"/>
+                <param name="threshold_min" value="8"/>
+                <param name="threshold_max" value="255"/>
+            </conditional>
+            <param name="display" value="bw"/>
+            <output name="output" file="blobs_threshold_8-255.gif" compare="sim_size"/>
+        </test>
+        <test>
+            <param name="input" value="blobs.gif"/>
+            <conditional name="method_cond">
+                <param name="auto_method" value="Manual"/>
+                <param name="threshold_min" value="0"/>
+                <param name="threshold_max" value="8"/>
+            </conditional>
+            <param name="display" value="bw"/>
+            <output name="output" file="blobs_threshold_0-8.gif" compare="sim_size"/>
+        </test>
+        <test>
+            <param name="input" value="confocal-series-first-channel.tif"/>
+            <conditional name="method_cond">
+                <param name="method" value="Default"/>
+                <param name="black_background" value="yes"/>
+            </conditional>
+            <param name="display" value="bw"/>
+            <output name="output" file="confocal-series-first-channel_threshold_default.tiff" compare="sim_size"/>
```

Comment:
It would be nice to use the new fancy imaging asserts from @kostrykin  here.

https://docs.galaxyproject.org/en/latest/dev/schema.html#has-image-center-of-mass